doc/bootstrap.rst

```diff
@@ -6,22 +6,35 @@ A new Ceph cluster is deployed by bootstrapping a cluster on a single
 node, and then adding additional nodes and daemons via the CLI or GUI
 dashboard.
 
+In the example here, we will install a basic three-node cluster. Each
+node will be identified by its prompt. For example, "[monitor 1]"
+identifies the first monitor, "[monitor 2]" identifies the second
+monitor, and "[monitor 3]" identifies the third monitor. This
+information is provided in order to make clear which commands
+should be issued on which systems.
+
+"[any monitor]" identifies any Ceph node, and in the context
+of this installation guide means that the associated command
+can be run on any node.
+
 Get ceph-daemon
 ===============
 
 The ``ceph-daemon`` utility is used to bootstrap a new Ceph Cluster.
-You can get the utility by either installing a package provided by
-your Linux distribution::
 
-  sudo apt install -y ceph-daemon   # or,
-  sudo dnf install -y ceph-daemon   # or,
-  sudo yum install -y ceph-daemon
+Use curl to fetch the standalone script::
+
+  [monitor 1] # curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/ceph-daemon/ceph-daemon
+  [monitor 1] # chmod +x ceph-daemon
+  [monitor 1] # sudo install -m 0755 ceph-daemon /usr/sbin    # optional!
+  
+You can also get the utility by installing a package provided by
+your Linux distribution::
 
-or by simply downloading the standalone script manually::
+   [monitor 1] # sudo apt install -y ceph-daemon   # or,
+   [monitor 1] # sudo dnf install -y ceph-daemon   # or,
+   [monitor 1] # sudo yum install -y ceph-daemon
```

Comment:
Let's add as well:
[monitor 1] # sudo zypper install -y ceph-daemon

Reply:
I have added a zypper command to this list.

I've also taken the opportunity here to remove the sudos, since the prompts are clearly privileged prompts and signal the level of elevated privilege from which the commands must be issued.

Reply:
I would have thought that Sphinx's inability easily to escape hashes, given that hashes designate root prompts in Linux and given that such prompts can reasonably be expected to occur with regularity in Linux documentation, would have provoked an outcry from every corner of the earth so loud and pervasive as to drown out every other conversation until the sorry state of affairs that provoked the outcry was addressed and remedied, but no. It hasn't. And so here we are, powerless to remedy this easily, stuck with documentation that we know includes wrongly italicized lines.

Anyway, the zypper command has now been added to the documentation, and the installation commands are not escaped because we are imprisoned in a world defined by the limitations of reStructuredText.